e2e-tests/tools/daemon-util/index.js

```diff
@@ -39,6 +42,15 @@ function startNpmScript(script, color) {
 
   child.on('exit', () => {
     console.log(normal(`Finished "npm run ${script}"`));
+    // This is needed to kill the node server process on windows which is not automatically killed
+    if(platform.os.family === 'Win32') {
+      cmd.get(
+        'TASKKILL /F /IM node.exe',
```

Comment:
Seems like this will terminate all `node` instances. We should be able to get the `pid` from the [child process](https://nodejs.org/api/child_process.html#child_process_subprocess_pid) and terminate it by `pid`.

Reply:
@jmelberg-okta 
I tried to get the child process pid and terminate using that. But it wasn't successful. This is what I observed -> 
* Since I set `spawnWith: {shell: true}`, a new shell is spawned (which is the child process of the original npm test script that we run)
* Now this new child shell spawns the node process which starts the server -> This is the node process that needs to be killed
* But this node process is a grandchild of the original test script and there is no easy way to get the pid of the grand child process
* The only way I could think was to stop all node processes

Reply:
This is probably fine for our use cases. Thanks for taking a look